### ISSUE TITLE
Experience drops no longer centered when set to Attach to player

### ISSUE THREAD

**Issue by Miqote-Petter** [l2-/template-plugin#20]
Experience drops, which used to be centered on the character, are now offset significantly to the right, despite both X and Y axis "attach to offset" set to zero.

In the image below, it is now over the spellbook. 

![image](https://user-images.githubusercontent.com/102330759/160019633-d697d9b1-63a9-4e81-9102-820e36c40d5b.png)

**Comment by l2-** [comment]
Does it occur consistently? I cannot seem to reproduce this.

**Comment by Miqote-Petter** [comment]
> Does it occur consistently? I cannot seem to reproduce this.

It does. Is there any logs that I might be able to give to you?

**Comment by l2-** [comment]
Nothing is logged that would help me figure this out I think. Does the xp drop move with relation to the player as in it is clearly attached to the player just offset by a lot? Is the position of the overlay correct when you do not attach to player? what about attach to npc?

**Comment by Miqote-Petter** [comment]
Hi there, sorry. I assumed replying to the email -- since it says I can -- would post it here, but I guess not.

Yes, it stays in the same spot and acts like it is simply offset -- if I set an offset of around -1400 or so, it is centered for me, but that's only when I have it maximized. 

I have tried resetting the mod, too.

I am able to live with it, however, and am quite appreciative of your work on this mod. It is nice having the option to change the location of where the xp drop is.
